### DISCUSSION TITLE
Command Fail Seek: Track 0 Not Found

### DISCUSSION THREAD

**Discussion by NintenMSDOS** [keirf/greaseweazle#223] [Q&A]
Hello, sorry if this questions seems kinda dumb, but nobody got my problem, and searching online no one show how to fix this issue. 

I have a Greaseweazle F1. It is working, the command gw -info shows the complete and correct status of my Greaseweazle. I am trying to write some amiga .adf images to my floppy disk (which are 2HD regular IBM ones, **with the hole covered of course**). I follow tutorials and guides on how to write the amiga adf. but when I try to write I get that message.

_Command Fail Seek: Track 0 Not Found_

I tried changin the Drive select, which was set on "A" (as the Wiki says may be the issue) but is not. Got the same error. Or drive not found. (Because the drive in the greaseweazle case is a PC Floppy).

Trying to erase the disk also gets the same error. The floppy disk are new. So I'm wondering what I'm doing wrong. Since i'm doing the same steps I've seen on internet/youtube guides about how to write amiga .adf files. 
Hope someone can help me.

**Comment by keirf** [comment]
Is this using an F1 adapter for the blue pill board?

**Reply by NintenMSDOS** [reply]
Is the blue pill board. I show you a pic

https://imgur.com/a/rgicIVN

**Reply by keirf** [reply]
It's very hard to get reliable operation with direct dupont connections. Dupont connectors are not reliable. You will need to double check all connections and you may need ground connection to at least one of the floppy odd-numbered pins.

Really you need an F1 adapter board.

**Reply by NintenMSDOS** [reply]
I double checked the connections, I measure de USB bandwich (and is fine). There is one connected to ground. (no bliking led that show any problems) so? I don't know. Should I format the floppy before try to erase/write with the greazeweasel?

**Reply by keirf** [reply]
Trk 0 not found means that one of the following is bad: select, trk0 or step. You are not yet even trying to read or write data. One of these three is bad.

**Reply by NintenMSDOS** [reply]
There is no trk0 option in the GUI or GW commands.... Anyway I tried many differents configs and all ends in the same trak 0 not found. So maybe I should get DD Floppys instead of the HD ones.... (since in the tutorials/guide I followed work for them, but they used DD disk)

**Reply by keirf** [reply]
I mean that those physical signal connections must be suspected. Your problem is physical setup (or the drive is bad but in this setup a simple connect issue is likely). Are you sure select is connected to pin 12 and that this is a normal PC drive?

**Reply by NintenMSDOS** [reply]
Ok after many attemps, connect and reconnect cables, I can format/erase disk. But I cannot write to them. I Get  *FATAL ERROR* Fail to verify track 0.0

**Reply by keirf** [reply]
Can you read a known good disk?

**Reply by NintenMSDOS** [reply]
Forget about it, I could do it now. All tracks verified. My main problem I guess was (my greaseweazel has a plastic shell, the plastic shell closed must bends cables to a position it disconects. witouth the plastic shell I move the cables correctly, and tried to write without it. now it works. thanks a lot
